docs/time_series_causal_graph.md

```diff
@@ -0,0 +1,387 @@
+# Time Series Causal Graph
```

Comment:
If you want to represent a minimal graph, I can't comment on the image `docs/images/ts_minimal_graph.png` but doesn't seem correct. The edge `X1 lag(n=1) -> X2 lag(n=1)` should not be there.

If the purpose was different, please ignore my comment.